### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by chepkok3**
Added the footer section in the portfolio with the contact form html tag.
Added the css styling for the contact form to enhance the appearance of the footer section and its alignment.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -315,6 +315,62 @@ <h2 class="connect">LET'S CONNECT</h2>
...
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  required
+    
+                  type="text"
+                  cols="30"
+                  rows="10"
+                ></textarea>
+              </div>
+
```
Comment (ichala) [inline index.html:364]: - [ ] kindly fix those design issues to match Figma template : 

- Kindly Remove the red outline/border from your form 
- Kindly remove the empty space from the left side


image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @chepkok3 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Validation Works  ✔️ 
-  Good design ✔️
- No Linter errors ✔️


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @chepkok3, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] Hi @chepkok3, well done implemented the changes requested by the previous reviewer 👍 👍 👍, but while making the effort to get the extra spaces around your contact section fixed, i noticed you introduced another error in to your work. I would kindly suggest you fix the extra white space at the bottom of your contact section

Here is a picture to that effect 👇 

alor

- You can rectify this by checking the styling you gave your `.section-grid`.

Change it from this: 👇 
```CSS
.section-grid {
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: 1fr;
  grid-gap: 88px;
  padding: 114px 24px;
}
```

To: 👇 

```CSS
.section-grid {
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: 1fr;
  grid-gap: 88px;
  padding: 0 24px;
}
```

> Note: I would have added these comments with GH inline comments but because you worked on the issues raised in previous milestone, I couldn't add inline comments to the concerned changes. Kindly go through the points highlighted above. Thanks. 👍

- [x]  _Please check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] I marked this as optional because it wasn't part of the requirement for this `contact section` of the project but i will urge you to kindly work on it. the `About section` of your project looks different from what we have in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=lL507P5kPHpySbVe-0) and i would recommend you to kindly check it and fix it.


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi @chepkok3, 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**